test/dummy/app/versions/orange_version.rb

```diff
@@ -0,0 +1,3 @@
+class OrangeVersion < PaperTrail::Version
+  self.table_name = "orange_versions"
+end
```

Comment:
The `orange_versions` table is necessary because you need a table whose `item_id` column is a string, right?